.gitignore

```diff
@@ -1,5 +1,4 @@
 .vagrant
-Berksfile.lock
```

Comment:
I intended to un-ignore it. Both Berksfile.lock and Gemfile.lock need to be part of the repo to make some of the testing components operate correctly. There is some debate on whether to include them or not; including the lock files solidifies the testing to known working versions.